indexers/vector/FaissIndexer/__init__.py

```diff
@@ -130,6 +130,9 @@ def query(self, vecs: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.nd
             from faiss import normalize_L2
             normalize_L2(vecs)
         dist, ids = self.query_handler.search(vecs, top_k)
+        if self.distance == 'inner_product':
+            self.logger.warning('inner_product will be output as distance.')
```

Comment:
This warning should rather go into the `build_advanced_index` method. Logging it with every request is overkill, in my opinion.